router/src/lib.rs

```diff
@@ -63,3 +63,29 @@ pub type RouterRef = Arc<dyn Router + Sync + Send>;
 pub trait Router {
     async fn route(&self, req: RouteRequest) -> Result<Vec<Route>>;
 }
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RouteCacheConfig {
+    /// Time to live (TTL) in second
+    ttl: u64,
+    /// Time to idle (TTI) in second
+    tti: u64,
+    /// how many route records can store in cache
+    capacity: u64,
+}
+
+impl RouteCacheConfig {
+    pub fn new() -> Self {
+        Self {
+            ttl: 5 * 60,
+            tti: 2 * 60,
+            capacity: 10_000,
+        }
+    }
+}
```

Comment:
```suggestion
        Self::default()
    }
}
```

Reply:
> 

This will make the unit test difficult to write.